dependencyManagement/build.gradle.kts

```diff
@@ -72,7 +72,7 @@ val DEPENDENCIES = listOf(
   "commons-logging:commons-logging:1.2",
   "com.sparkjava:spark-core:2.9.3",
   "com.squareup.okhttp3:okhttp:4.9.3",
-  "io.opentelemetry.contrib:opentelemetry-aws-xray:1.12.0",
+  "io.opentelemetry.contrib:opentelemetry-aws-xray:1.13.0",
   "io.opentelemetry.proto:opentelemetry-proto:0.11.0-alpha",
```

Comment:
shall we update this to 1.16. or unless any compatibility/stablness we have at 1.11

https://search.maven.org/artifact/io.opentelemetry.proto/opentelemetry-proto/0.16.0-alpha/jar

Reply:
DO NOT update this to `0.15.0` or beyond without confirming that the corresponding exporter has been updated to use `InstrumentationScope` instead of `InstrumentationLibrary`.  Failure to do so can cause lost telemetry.